zgw/idtoken-authentication/src/test/kotlin/com/ritense/portal/idtokenauthentication/service/IdTokenGeneratorTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.ritense.portal.idtokenauthentication.service
+
+import io.jsonwebtoken.Jwts
+import kotlin.test.assertEquals
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+
+class IdTokenGeneratorTest {
+    private val idTokenGenerator = IdTokenGenerator()
+
+    @Test
+    fun `should generate token`() {
+        val testSecretKey = "NYy.TAQYbHRRWC_b2rfoqs4oR9jXj38N"
```

Comment:
Can we make it obvious that this is a fake secret? It looks like a real one, although I guess it's not.

Reply:
the variable name is clear enough?

but I guess I can move it somewhere and make it an obvious test constant.